Gopkg.toml

```diff
@@ -150,7 +150,7 @@ required = [
 
 [[constraint]]
   name = "github.com/Shopify/sarama"
-  version = "1.20.1"
```

Comment:
Is there a way to depend on a version?

Reply:
Hopefully a new release should be coming soon: https://github.com/Shopify/sarama/pull/1366#issuecomment-496850147

Reply:
This is just temporal, waiting for a new release of sarama.

Reply:
@pavolloffay if we don't get a sarama release soon, are we ok to release with the SHA?